config/files/hyprland/usr/etc/xdg/waybar/config

```diff
@@ -5,26 +5,26 @@
     // "width": 1280, // Waybar width
     "spacing": 4, // Gaps between modules (4px)
     // Choose the order of the modules
-    "modules-left": ["sway/workspaces", "sway/mode", "sway/scratchpad", "custom/media"],
-    "modules-center": ["sway/window"],
-    "modules-right": ["mpd", "idle_inhibitor", "pulseaudio", "network", "cpu", "memory", "temperature", "keyboard-state", "sway/language", "battery", "battery#bat2", "clock", "tray"],
```

Comment:
please move the entire hyprland/usr/etc/xdg dir from config/files/hyprland to config/files/usr

so this file for example should be in config/files/usr/etc/xdg/waybar/config

Reply:
Also moved these.